models/lobby.go

```diff
@@ -14,8 +13,8 @@ type Whitelist int
 type LobbyState int
 
 const (
-	LobbyTypeSixes      LobbyType = 0
-	LobbyTypeHighlander LobbyType = 1
+	LobbyTypeSixes      LobbyType = 6
```

Comment:
I don't like this. We won't be able to have two different types that have the same number of players.